dask-gateway/dask_gateway/dask_cli.py

```diff
@@ -576,6 +583,8 @@ async def start_worker(
     scheduler_address=None,
     local_directory="",
     nanny=True,
+    dashboard=True,
```

Comment:
The default of `True` here matches the `dask-worker` CLI default. It differs from the `distributed.Worker` default of False. I think following the CLI default is appropriate.